utils/githooks/pre-commit.d/40-pylint.sh

```diff
@@ -10,10 +10,13 @@ echo "Pylint:"
 # shellcheck disable=SC1091
 . utils/githooks/find_base.sh
 
-if [ "$TARGET" = "HEAD" ]; then
-    echo "  Checking against HEAD"
-    git diff HEAD --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
-else
-    echo "  Checking against branch ${TARGET}"
-    git diff "$TARGET"... --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
+
+if [ -f utils/cq/daos_pylint.py ]; then
+    if [ "$TARGET" = "HEAD" ]; then
+            echo "  Checking against HEAD"
+            git diff HEAD --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
+    else
+            echo "  Checking against branch ${TARGET}"
+            git diff "$TARGET"... --name-only | ./utils/cq/daos_pylint.py --files-from-stdin
+    fi
```

Comment:
Same comment for here.  On this branch the files tested for do exist.

Reply:
Again, it is likely due to trying to use the same githooks in other repos.